code/modules/station_goals/bluespace_tap.dm

```diff
@@ -0,0 +1,359 @@
+							/obj/item/stock_parts/capacitor/quadratic = 5,//Probably okay, right?
+							/obj/item/stack/ore/bluespace_crystal = 5)
+
+/obj/effect/spawner/lootdrop/bluespace_tap
+	name = "bluespace tap reward spawner"
+	lootcount = 1
+	color = "#00FFFF"
+
+/obj/effect/spawner/lootdrop/bluespace_tap/hat
+	name = "a head covering"
+	loot = list(
+			/obj/item/clothing/head/collectable/chef,	//same weighing on all of them
+			/obj/item/clothing/head/collectable/paper,
+			/obj/item/clothing/head/collectable/tophat,
+			/obj/item/clothing/head/collectable/captain,
+			/obj/item/clothing/head/collectable/beret,
+			/obj/item/clothing/head/collectable/welding,
+			/obj/item/clothing/head/collectable/flatcap,
+			/obj/item/clothing/head/collectable/pirate,
+			/obj/item/clothing/head/collectable/kitty,
+			/obj/item/clothing/head/crown/fancy,
+			/obj/item/clothing/head/collectable/rabbitears,
+			/obj/item/clothing/head/collectable/wizard,
+			/obj/item/clothing/head/collectable/hardhat,
+			/obj/item/clothing/head/collectable/HoS,
+			/obj/item/clothing/head/collectable/thunderdome,
+			/obj/item/clothing/head/collectable/swat,
+			/obj/item/clothing/head/collectable/slime,
+			/obj/item/clothing/head/collectable/police,
+			/obj/item/clothing/head/collectable/slime,
+			/obj/item/clothing/head/collectable/xenom,
+			/obj/item/clothing/head/collectable/petehat
+	)
+
+
+/obj/effect/spawner/lootdrop/bluespace_tap/cultural
+	name = "cultural artifacts"
+	loot = list(
+		/obj/vehicle/space/speedbike/red = 10,
+		/obj/item/grenade/clusterbuster/honk = 10,
+		/obj/item/toy/katana = 10,
+		/obj/item/stack/tile/brass/fifty = 20,
+		/obj/item/stack/sheet/mineral/abductor/fifty = 20,
+		/obj/item/sord = 20,
+		/obj/item/toy/syndicateballoon = 15,
+		/obj/item/lighter/zippo/gonzofist = 5,
+		/obj/item/lighter/zippo/engraved = 5,
+		/obj/item/lighter/zippo/nt_rep = 5,
+		/obj/item/gun/projectile/automatic/c20r/toy = 1,
+		/obj/item/gun/projectile/automatic/l6_saw/toy = 1,
+		/obj/item/gun/projectile/automatic/toy/pistol = 2,
+		/obj/item/gun/projectile/automatic/toy/pistol/enforcer = 1,
+		/obj/item/gun/projectile/shotgun/toy = 1,
+		/obj/item/gun/projectile/shotgun/toy/crossbow = 1,
+		/obj/item/gun/projectile/shotgun/toy/tommygun = 1,
+		/obj/item/gun/projectile/automatic/sniper_rifle/toy = 1,
+		/obj/item/twohanded/dualsaber/toy = 5,
+		/obj/machinery/snow_machine = 10,
+		/obj/item/clothing/head/kitty = 5,
+		/obj/item/coin/antagtoken = 5,
+		/obj/item/toy/prizeball/figure = 15,
+		/obj/item/toy/prizeball/therapy = 10,
+		/obj/item/bedsheet/patriot = 2,
+		/obj/item/bedsheet/rainbow = 2,
+		/obj/item/bedsheet/captain = 2,
+		/obj/item/bedsheet/centcom = 1, //mythic rare rarity
+		/obj/item/bedsheet/syndie = 2,
+		/obj/item/bedsheet/cult = 2,
+		/obj/item/bedsheet/wiz = 2
+	)
+
+/obj/effect/spawner/lootdrop/bluespace_tap/organic
+	name = "organic objects"
+	loot = list(
+		/obj/item/seeds/random/labelled = 50,
+		/obj/item/guardiancreator/biological = 5,
```

Comment:
is this the swarm holopara? seems like this should be a bit rarer

Reply:
It has a 5 in 168 chance of appearing, so roughly 3%, same for the x-ray injector. With current point balance you can afford to buy this maybe 3 times total if focusing on that and not doing anything super-crazy with power. I think 3 shots at 3% each at a holopara is sufficiently rare. Same for the x-ray.